src/estimation/include/iDynTree/Estimation/ContactStateMachine.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Silvio Traversaro, Prashanth Ramadoss
+ * email: silvio.traversaro@iit.it, prashanth.ramadoss@iit.it
+ *
+ * Permission is granted to copy, distribute, and/or modify this program
+ * under the terms of the GNU General Public License, version 2 or any
+ * later version published by the Free Software Foundation.
+ *
+ * A copy of the license can be found at
+ * http://www.robotcub.org/icub/license/gpl.txt
```

Comment:
I just noticed that this is a dead URL. Anyhow, I saw that you copied the header from other files, so it probably make more sense to change all the headers once.

Reply:
I did not check the contents of the url! I just copy-pasted, yes ! What do you suggest to do about this ?

Reply:
Leave it as that, we will cleanup the headers automatically in the future.